src/composables/use-ec-sorting/use-ec-sorting.ts

```diff
@@ -0,0 +1,50 @@
+import { ref, unref } from 'vue';
+
+import { SortDirection, SortDirectionCycle } from '../../enums';
+import type { SortingOptions } from './types';
+
+export default function useEcSorting({
+  initialSorts = [],
+  isMultiSort = false,
+  sortCycle: sortCycleInOptions,
+}: Partial<SortingOptions> = {}) {
```

Comment:
I think we can add `?` to type definition in types file and remove Partial from here. As now if I look at that type I see that all fields are required, but they are definitely not.